pages/dkp/konvoy/2.1/storage/static-local-volume/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: layout.pug
+navigationTitle: Provision a static local volume
+title: Provision a static local volume
+menuWeight: 15
+excerpt: Learn how to provision a static local volume for a Konvoy cluster
+beta: false
+enterprise: false
+---
+
+The `localvolumeprovisioner` component uses the [local volume static provisioner][localstorage] to manage persistent volumes for pre-allocated disks.
+It does this by watching the `/mnt/disks` folder on each host and creating persistent volumes in the `localvolumeprovisioner` storage class for each disk that it discovers in this folder.
+
+-   Persistent volumes with a 'Filesystem' volume-mode are discovered if they are mounted under `/mnt/disks`.
+
+-   Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
+
+## Before you begin
+
+Before starting this tutorial, verify the following:
+
+-   You have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+-   You have a provisioned `konvoy` cluster that uses the `localvolumeprovisioner` component, but have not added any other Kommander applications to the cluster yet.
+
+For this tutorial, you **do not deploy** using all the default settings as described in the [Quick start guide][quickstart].
+
+This distinction between provisioning and deployment is important because some applications depend on the storage class provided by the `localvolumeprovisioner` component and can fail to start if it is not configured yet.
+
+### Provision the cluster and a volume
+
+1.  Create a pre-provisioned cluster by following the steps outlined [in the choose pre-provisioned infrastructure topic][preprovision].
+
+    Whenever volumes are create on the node machines, the local volume provisioned detects the volume in `/mnt/disks` and adds it as a persistent volume with the `localvolumeprovisioner` storage class.
+
+1.  Create some volumes in `/mnt/disks` on each host.
```

Comment:
Is more than one required? If not, we may want to write, "Create at least one volume in...."

If more than one **is** required, how many do I have to have?

Reply:
@dkoshkin / @faiq  can you confirm the requirements with this?

Reply:
It is required if the user wants that node to run workloads with a volume. 

So, I think the "Create at least one filesystem on each host"